components/brave_wallet_ui/utils/network-utils.test.ts

```diff
@@ -29,3 +38,33 @@ describe('reduceNetworkDisplayName', () => {
     expect(reduceNetworkDisplayName(networkName)).toBe(expected)
   })
 })
+
+describe('getNetworksByCoinType', () => {
+  it('CoinType ETH, should return all ETH networks', () => {
+    expect(getNetworksByCoinType(mockNetworks, BraveWallet.CoinType.ETH)).toEqual(mockNetworks)
+  })
+  it('CoinType random number, should return an empty array', () => {
+    expect(getNetworksByCoinType(mockNetworks, 3000)).toEqual([])
+  })
+})
+
+describe('getTokensNetwork', () => {
+  it('Ethereum with chainId 0x1, should return ETH Mainnet info', () => {
+    expect(getTokensNetwork(mockNetworks, NewAssetOptions[0])).toEqual(mockNetworks[0])
```

Comment:
Can we alias NewAssetOptions[0] & mockNetworks[0] to a variable with a more descriptive name?

Reply:
Done!